src/components/molecules/DoubleBarChartView.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from 'react';
+import { ResponsiveContainer, BarChart, LabelList, XAxis, Bar } from 'recharts';
+import { DoubleBarChartFillColorBar1, DoubleBarChartFillColorBar2 } from '../../style';
+
+interface IProps {
+  data: Array<object>;
+  yLabel: string | number;
+  xLabel1: string | number;
+  xLabel2: string | number;
+  textLabel?: string;
+}
+const DoubleBarChartView: FC<IProps> = ({ data, xLabel1, xLabel2, yLabel }) => {
+  return (
+    <ResponsiveContainer width={'100%'} height={'70%'}>
```

Comment:
If possible, make container take the entire space, and determine the size from outside the component.
If not - `height` should be provided as a `prop`